File: leonardsangoroh/Arrays and Strings/MinimumSwaps.java
package gol;

/**
 *
 * @author leesangoroh
 */
public class MinimumSwaps {
    
    static int minimumSwaps (int [] array) {
        
        int count=0;
        int temp;
        int length = array.length -1;
        
        for (int i = 0; i<length; i++) {
            
            //if (array[i] < array[i++] || array[i] == array[i++]) {
               
              //  continue;
            //}
            
            if (array[i]>array[i++]) {
                temp = array[i];
                array[i] = array[i++];
                array[i++] = temp;
                
                count++;
            }
        }
        
        return count;
    }
    
    
    public static void main (String [] args) {
        
        int [] arr = {5,7,1,3,0,8,6};
        
        System.out.println(minimumSwaps(arr));
    }
}

File: AugustineKadima/Arrays_And_Strings/minimumSwaps.js

const minimumSwaps = (arr) => {
    let n = arr.length;
    let i;
    let count = 0;
    for(i = 0; i < n; i++){
        const temp = arr[i + 1];
        const temp2 = arr[n-1]
        if(arr[i] > arr[i + 1]){
            arr[i + 1] = arr[i];
            arr[i] = temp;
            count++;
        }
       
    }

    for(i = 0; i < n; i++){
        for(let j = 0; j < n; j++){
            let temp = arr[j]
          if(arr[j] > arr[j + 1]){
              arr[j] = arr[j + 1]
              arr[j + 1] = temp
              count++
          }
        }
    }


    console.log(count);
    console.log(arr);

}

minimumSwaps([4,2,56,7,1,9])